Dockerfile

```diff
@@ -1,17 +1,19 @@
 # Stage-1
 # Build jar
-
 FROM gradle:6.7.0-jdk15 as builder
 LABEL maintainer="Johannes Lichtenberger <johannes.lichtenberger@sirix.io>"
 WORKDIR /usr/app/
 
+# Basic environment variables
+ENV Xms 3g
+ENV XMX 8g
+
```

Comment:
Die werden nirgends ausgelesen, oder? Ich würds auch eher JAVA_OPTS benennen und dann alles in eine environment Variable packen :)

Reply:
alrighty!

Reply:
Und im CMD-Command noch als Argument mit rein, oder?

Reply:
Nimmer er das nicht automatisch? Ist das tomcat spezifisch, dass das automatisch funktioniert?

Reply:
Tatsache, ist nur beim Tomcat automatisch...